### DISCUSSION TITLE
Custom error for inactive users?

### DISCUSSION THREAD

**Discussion by leothomas** [fastapi-users/fastapi-users#1453] [Q&A]
It looks like `fastapi_users` raises a generic `Exception` when an inactive user attempts to login ([src](https://github.com/fastapi-users/fastapi-users/blob/f37a48f97d6c68384ffc5060808e655741156a88/fastapi_users/exceptions.py#L20)). 

Is it possible to returns a specific error code or custom error message for inactive users? This would allow us to differentiate this scenario from a generic error with the registration/login procedure and provide more information to the user.

I couldn't find anything about customizing error codes/messages in the docs - please feel free to point me in the right direction if it exists! 

Thanks y'all!

**Comment by hgalytoby** [comment]
For now, we can only rewrite the login router

https://github.com/fastapi-users/fastapi-users/blob/master/fastapi_users/router/auth.py#L59
